docs/introduction/introduction.md

```diff
@@ -73,19 +95,19 @@ High Level Architecture
 
 ### Services
 
-- Parsing - normalising logs into messages with one layer of key/value pairs
-- Enrichment - adding useful data to events to assist in detection and investigations
-- Alerting - filtering matching events from an incoming data stream of events based on a configurable rule set. The correlation alerting allows one to group several detections together before raising an alert
-- Response
+- Parsing - normalizing logs in messages with one layer of key/value pairs.
+- Enrichment - adding useful data to events to assist in detection and investigations.
+- Alerting - filtering matching events from an incoming data stream of events based on a configurable rule set. The correlation alerting allows users to group several detections together before raising an alert.
+- Rapid Response - flexible incident response workflows can be built and triggered in real-time via the highly modular and pluggable framework.
```

Comment:
Why 'Rapid' was added to the siembol response service name?

Reply:
"Rapid Response" is language taken from the Siembol.io website.

Reply:
Ok. Meybe it can be confusing to use feature names from web while siembol response is the name of the component used in the codebase including docs.